src/term/display.rs

```diff
@@ -67,6 +67,7 @@ impl Term {
       }
       Term::Era => write!(f, "*"),
       Term::Num { val } => write!(f, "{val}"),
+      Term::Str { val } => write!(f, "{val}"),
```

Comment:
```suggestion
      Term::Str { val } => write!(f, "{val:?}"),
```
(to show it with quotes/escapes)